bo/ansprechpartner.go

```diff
@@ -15,13 +15,15 @@ type Ansprechpartner struct {
 	Titel              titel.Titel          `json:"titel,omitempty"`                                    // Titel ist ein möglicher Titel des Ansprechpartners
 	Vorname            string               `json:"vorname,omitempty"`                                  // Vorname ist der Vorname des Ansprechpartners
 	Nachname           string               `json:"nachname,omitempty" validate:"required"`             // Nachname ist der Familienname des Ansprechpartners
+	Email              string               `json:"eMailAdresse,omitempty"`                             // Email-Adresse des Ansprechpartners
```

Comment:
```suggestion
	Email              string               `json:"eMailAdresse,omitempty" validate:"omitempty,email"`                             // Email-Adresse des Ansprechpartners
```
https://pkg.go.dev/github.com/go-playground/validator/v10#section-readme